controllers/manager/manager.go

```diff
@@ -197,6 +198,12 @@ type PlatformManager struct {
 	GetPlatformClientImpl           func(namespace string) *gophercloud.ServiceClient
 }
 
+type RestoreStatus struct {
+	InSync           *bool   `json:"inSync"`
+	DeploymentScope  *string `json:"deploymentScope"`
```

Comment:
I would suggest to update the scope to boostrap and required to not_required, mainly because we should not let DM to do anything during the restore period

Reply:
removed deploymentScope and strategyRequired from struct and backup shell script and had assigned them to "bootstrap" and "not_required" in DM